packages/swing-store/src/transcriptStore.js

```diff
@@ -109,6 +140,31 @@ export function makeTranscriptStore(
     ON transcriptSpans (vatID, isCurrent)
   `);
 
+  const sqlReadCompressedSpanItems = db.prepare(`
+    SELECT items
+    FROM transcriptCompressedSpans
+    WHERE vatID = ? AND startPos = ?
+  `);
+  sqlReadCompressedSpanItems.pluck();
+
+  function readCompressedSpan(vatID, startPos) {
+    const blob = sqlReadCompressedSpanItems.get(vatID, startPos);
+    if (!blob) {
+      return undefined;
+    }
+    const items = gunzipSync(blob).toString().split('\n');
+
+    function* reader() {
+      for (const item of items) {
+        if (item !== '') {
```

Comment:
In what case would an item be empty?

Reply:
There are no empty items per se, but since our convention is that they're newline terminated rather than newline separated means splitting the items string at the newlines will always result in an extra empty "item" at the end.